araproc/analysis/standard_reco.py

```diff
@@ -452,19 +468,26 @@ def __get_correlation_function(self, ch1, ch2, wavepacket):
           self.__latest_event_num = event_number
 
           # get the correlation functions
-          self.__corr_functions_v = self.__calculate_cross_correlations(waveform_bundle, self.pairs_v)
-          self.__corr_functions_h = self.__calculate_cross_correlations(waveform_bundle, self.pairs_h) 
+          self.__corr_functions_v_fullphase = self.__calculate_cross_correlations(waveform_bundle, self.pairs_v, False)
+          self.__corr_functions_h_fullphase = self.__calculate_cross_correlations(waveform_bundle, self.pairs_h, False)
+          self.__corr_functions_v = self.__calculate_cross_correlations(waveform_bundle, self.pairs_v, True)
+          self.__corr_functions_h = self.__calculate_cross_correlations(waveform_bundle, self.pairs_h, True) 
 
         if(ch1//8 != ch2//8):
```

Comment:
Not a part of this PR, but should this check channel polarization directly or do we never expect to put PA data through this framework?

Reply:
No PA won't ever be put thru AraProc, but it's a good point. There's a better way to do this. I will update.

Reply:
Cool! There are a couple of other spots that use this notation that I didn't flag.

Reply:
Oh you're right. I think I got them all now. Thanks!